atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/UniqueSchemaMutationLockTable.java

```diff
@@ -84,11 +85,28 @@ private TableReference getSingleTable() throws TException {
         Set<TableReference> lockTables = schemaMutationLockTables.getAllLockTables();
 
         if (lockTables.size() > 1) {
-            throw new IllegalStateException("Multiple schema mutation lock tables have been created.\n"
-                    + "This happens when multiple nodes have themselves as lockLeader in the configuration.\n"
-                    + "Please ensure the lockLeader is the same for each node, stop all Atlas clients using this "
-                    + "keyspace, restart your cassandra cluster and run the clean-cass-locks-state cli command.\n"
-                    + "The tables that clashed were: " + lockTables);
+            // Try to recover by checking if other lock tables are empty and dropping them.
+
+            Set<TableReference> emptyLockTables = lockTables.stream()
+                    .filter(schemaMutationLockTables::isLockTableEmpty)
```

Comment:
Isn't this potentially risky? We could see the empty table, but then another node could fill the table before we get round to dropping it. What happens in that case?

Reply:
Yep. There were more edge cases around this, since propagating the creation of a table among the cassandra nodes is done asynchronously. I've removed this, since it seems to create more edge cases and is trying to solve a problem that has not happened in the field - checked our logs, and creating a lock table twice doesn't seem to happen often.

Reply:
I'm not surprised it doesn't happen often - lock table creation happens only when we start up for the first time, and double creation would mean someone set up their config badly (overriding the sensible defaults) AND hit the race condition.